Task_HelloWorld/index.css

```diff
@@ -1 +1,7 @@
 /* Write your code below for HELLO WORLD TASK*/
```

Comment:
The opening and closing tags should be consistent. Currently, the opening tag has an uppercase 'H' while the closing tag has a lowercase 'h'.